internal/cli/cmd/sysdump.go

```diff
@@ -102,7 +102,7 @@ func newCmdSysdump() *cobra.Command {
 		"Whether to enable quick mode (i.e. skip collection of 'cilium-bugtool' output and logs)")
 	cmd.Flags().IntVar(&sysdumpOptions.WorkerCount,
 		"worker-count", sysdump.DefaultWorkerCount,
-		"The number of workers to use")
+		"The number of workers to use\nNOTE: If a small value is specified, the actual number of workers may be adjusted automatically")
```

Comment:
I'd rephrase this slightly, maybe something like the following:

> NOTE: There is a lower bound requirement on the number of workers for the sysdump operation to be effective. Therefore, for low value, the actual number of workers may be adjusted upwards.

Reply:
Sure, makes sense!